src/Umbraco.Infrastructure/Migrations/MigrationPlanExecutor.cs

```diff
@@ -135,6 +144,12 @@ public ExecutedMigrationPlan ExecutePlan(MigrationPlan plan, string fromState)
             RebuildCache();
         }
 
+        // If any completed migration requires us to sign out the user we'll do that.
+        if (_invalidateBackofficeUserAccess)
+        {
+            RevokeBackofficeTokens().GetAwaiter().GetResult(); // should async all the way up at some point
```

Comment:
I myself am not happy about this.
Had a quick look up the chain and it seems doable to change everything to async but I might have missed a branch where it isn't

Reply:
Glad you agree with my stomach turning over thinking about all the signature changes 😄